src/reference/asciidoc/whats-new.adoc

```diff
@@ -17,5 +17,12 @@ In general the project has been moved to the latest dependency versions.
 [[x6.2-new-components]]
 === New Components
 
+[[x6.2-debezium]]
+==== Debezium Inbound Channel Adapter
+
+The Debezium Spring Integration Debezium channel adapter is part of the `spring-integration-debezium` module.
+The Debezium Engine based Change Data Capture (CDC) channel adapter, allows capturing database change events, converting them into Messages and streaming those to the outbound channels.
```

Comment:
May be `allows receiving database`?
Otherwise double "Capture" in this sentence sounds a bit awkward.

Reply:
imo, capturing is better aligned with the CDC terminology